bugshaker/src/main/java/com/github/stkent/bugshaker/flow/email/FeedbackEmailFlowManager.java

```diff
@@ -30,221 +35,270 @@
+				toaster.toast(warningString);
+				logger.d(warningString);
+			}
+
+		}
+
+	};
+
+	private final OnClickListener reportBugClickListener = new OnClickListener() {
+		@Override
+		public void onClick(final DialogInterface dialog, final int which) {
+			final Activity activity = activityReferenceManager.getValidatedActivity();
+			final Context context = activity.getBaseContext();
+			final Toaster toaster = new Toaster(activity);
+			final Logger logger = new Logger(true);
+			if (activity == null) {
+				return;
+			}
+
+			if (shouldAttemptToCaptureScreenshot(activity)) {
+				if (EmailCapabilitiesProvider.canSendEmailsWithAttachments
+					(activityReferenceManager.getValidatedActivity().getPackageManager())) {
+					screenshotProvider.getScreenshotUri(activity)
+						.single()
+						.observeOn(AndroidSchedulers.mainThread())
+						.subscribeOn(AndroidSchedulers.mainThread())
+						.subscribe(new Subscriber<Uri>() {
+							@Override
+							public void onCompleted() {
+								// This method intentionally left blank.
+							}
+
+							@Override
+							public void onError(final Throwable e) {
+								final String errorString = "Screenshot capture failed";
+								toaster.toast(errorString);
+								logger.e(errorString);
+
+								logger.printStackTrace(e);
+
+								sendEmailWithoutScreenshot(context, activity);
+							}
+
+							@Override
+							public void onNext(final Uri uri) {
+								LogcatUtil.saveLogcatToFile(context);
+								sendEmailWithScreenshot(activity, uri, Uri.fromFile(LogcatUtil.getLogFile()));
+							}
+
+
+						});
+				}
+				else {
+					sendEmailWithoutScreenshot(context, activity);
+				}
+			}
+			else {
+				final String warningString = "Window is secured; no screenshot taken";
+				toaster.toast(warningString);
+				logger.d(warningString);
+
+				sendEmailWithoutScreenshot(context, activity);
+			}
+		}
+	};
+
+	private void startActivity(Context applicationContext) {
+		File screenshotFile = ScreenshotUtil.getScreenshotFile(applicationContext);
+		String screenshotFileAbsolutePath = screenshotFile.getAbsolutePath();
+
+		if (screenshotFile.exists()) {
+			Intent goToMainActivityIntent = new Intent(applicationContext, MainActivity.class);
+			goToMainActivityIntent.putExtra("uri", screenshotFileAbsolutePath);
+			goToMainActivityIntent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+			applicationContext.startActivity(goToMainActivityIntent);
+		}
+		else {
+			throw new RuntimeException();
+		}
+	}
+
+	public void onActivityResumed(@NonNull final Activity activity) {
+		dismissDialog();
+		activityReferenceManager.setActivity(activity);
+	}
+
+	public void onActivityStopped() {
+		dismissDialog();
+	}
+
+	public void startFlowIfNeeded(Context context,
+		final boolean ignoreFlagSecure
+	) {
+		final Logger logger = new Logger(true);
+
+		if (isFeedbackFlowStarted()) {
+			logger.d("Feedback flow already started; ignoring shake.");
+			return;
+		}
+
+		this.ignoreFlagSecure = ignoreFlagSecure;
```

Comment:
if isWindowSecured is untrue, and ignoreFlagSecure is true, then the logger prints "Window is secure,d but we're ignoring that" 
if ignoreFlagSecure is true or isWindowSecured untrue, then shouldAttemptToCaptureScreenshot returns true